public/report/reportcard.js

```diff
@@ -42,7 +42,7 @@ $(function () {
     // Next button goes forward iff current block validates
     $('.form-navigation .next').on('click',function() {
         var cInd=curIndex();
-        if(cInd == 0 && !$('#inputGDPRAcknowledge').is(':checked')) {
+        if(cInd == 0 &&  (localStorage.getItem('username') && !$('#inputGDPRAcknowledge').is(':checked'))) {
```

Comment:
@qclin on the event modal this will still check for the check mark as localStorage.getItem('username') evaluates as true. I will soon push another commit soon that depends on modal / report card url instead.